examples/data/wflow_upgrade/sbm/wflow_sbm_v0x.toml

```
@@ -104,6 +104,9 @@ slope = "RiverSlope"
 width = "wflow_riverwidth"
 bankfull_depth = "RiverDepth"
 
+[input.lateral.river.floodplain.n]
+value = 0.144
+
 [input.lateral.subsurface]
 ksathorfrac = "KsatHorFrac"
```

I did not realise we did not have a value example for testing... The thing is that this particular model does not have floodplains so you can test with hydromt but not if the change does lead to a working Wflow.jl model. Instead I would update the ksathorfrac below (it is using a uniform map of value 100 anyway):

```suggestion
[input.lateral.subsurface.ksathorfrac]
value = 100
```